apps/dav/lib/Connector/Sabre/FilesPlugin.php

```diff
@@ -435,14 +437,31 @@ public function handleGetProperties(PropFind $propFind, \Sabre\DAV\INode $node)
 			});
 		}
 
-		if ($node instanceof \OCA\DAV\Connector\Sabre\Directory) {
+		if ($node instanceof Directory) {
 			$propFind->handle(self::SIZE_PROPERTYNAME, function () use ($node) {
 				return $node->getSize();
 			});
 
 			$propFind->handle(self::IS_ENCRYPTED_PROPERTYNAME, function () use ($node) {
 				return $node->getFileInfo()->isEncrypted() ? '1' : '0';
 			});
+
+			$requestProperties = $propFind->getRequestedProperties();
+			if (in_array(self::SUBFILE_COUNT_PROPERTYNAME, $requestProperties, true)
+				|| in_array(self::SUBFILE_COUNT_PROPERTYNAME, $requestProperties, true)) {
```

Comment:
```suggestion
				|| in_array(self::SUBFOLDER_COUNT_PROPERTYNAME, $requestProperties, true)) {
```

Also using `array_intersect` might be faster than the OR statement since most queries will execute both of the conditions.